addons/delivery/wizard/choose_delivery_package_views.xml

```diff
@@ -9,7 +9,7 @@
                     <field name="delivery_packaging_id"  domain="[('package_carrier_type', '=', context.get('current_package_carrier_type', False))]"
                       context="{'form_view_ref':'delivery.product_packaging_delivery_form'}"/>
                     <label for="shipping_weight" attrs="{'invisible': [('delivery_packaging_id', '=', False)]}"/>
-                    <div class="o_row" attrs="{'invisible': [('delivery_packaging_id', '=', False)]}">
+                    <div class="o_row" attrs="{'invisible': [('delivery_packaging_id', '=', False)]}" id="package_weight">
```

Comment:
use name="package_weight" instead of "id", the guidlines says its better. (even if it is not done everywhere :) )